src/dataset-builder/ConceptSelector.ts

```diff
@@ -102,11 +102,6 @@ export const ConceptSelector = (props: ConceptSelectorProps) => {
         headerStyle: tableHeaderStyle,
       }),
     ]),
-    cart.length !== 0 &&
-      h(ActionBar, {
-        prompt: cart.length === 1 ? '1 concept selected' : `${cart.length} concepts selected`,
-        actionText,
-        onClick: () => _.flow(onCommit)(cart),
-      }),
+    h(ConceptCart, { onClick: () => _.flow(onCommit)(cart), cart, actionText }),
```

Comment:
Is there a reason to have a different order of parameters here than in concept search? I know the order doesn't matter, but I think it's easier to see that they're the same if the order is the same.